e2e-app/src/app/datepicker/multiple/datepicker-multiple.component.html

```diff
@@ -0,0 +1,14 @@
+<h3>Datepicker focus tests</h3>
+
+<form>
+  <div>
+    <ngb-datepicker name="dp1" placeholder="yyyy-mm-dd" #d1 [startDate]="startDate"></ngb-datepicker>
```

Comment:
also `name`, `placeholder` and `#d1` are not necessary. Same for the second datepicker